lib/bot.js

```diff
@@ -119,15 +119,20 @@ Bot.prototype.run = function () {
     }
   });
   var prjNames = "";
-  _.each(self.config.projects, function (prj, index, list) {
-    pattern += prj;
-    prjNames += prj;
-    if (index != list.length - 1) {
-      pattern += "|";
-      prjNames += "|";
-    }
-  });
-  pattern += ")-[1-9][0-9]*)(\\+)?(?:(?=\\W)|$)";
```

Comment:
https://github.com/gsingers/slack-jira-plugin/pull/30 is the reason we have the leading [1-9] there.